### PR TITLE
instagram-challenge

### PR DESCRIPTION
**Pull request by jrose111284**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ApplicationHelper
```
Comment (yevdyko) [inline app/helpers/application_helper.rb]: Should be deleted if not needed.

File: app/helpers/posts_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module PostsHelper
```
Comment (yevdyko) [inline app/helpers/posts_helper.rb]: Should be deleted if not needed.

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  def index
+    @posts = Post.all
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
```
Comment (yevdyko) [inline app/controllers/posts_controller.rb]: If you are redirecting, you do not need to set @ variables

File: app/controllers/posts_controller.rb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def edit
+    @post = Post.find(params[:id])
+  end
+
+  def update
```
Comment (yevdyko) [inline app/controllers/posts_controller.rb]: If you are redirecting, you do not need to set @ variables

File: app/controllers/posts_controller.rb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    @post = Post.find(params[:id])
+    @post.update(post_params)
+
+    redirect_to '/posts'
+  end
+
+  def destroy
```
Comment (yevdyko) [inline app/controllers/posts_controller.rb]: If you are redirecting, you do not need to set @ variables

File: app/views/posts/edit.html.erb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<%= form_for @post do |f| %>
```
Comment (yevdyko) [inline app/views/posts/edit.html.erb]: Can be replaced with local variables.

**Comment by tansaku** [comment]
You had your instagram_challenge reviewed by **Pablo Domínguez**.
### The good points are:
* Correct installation instructions in README, Correct usage instructions in README
* Consistent indentation, Unnecessary files removed
* Text expectations appropriate, Test descriptions consistent, Tests DRY
* All controller methods short and succinct
* Thin controllers with minimal logic
* Model logic is DRY
* Relative and absolute paths used appropriately
* Expectations appropriately separated
* Responsive images supported with CSS
* Long nested call chains avoided

### You should consider the following improvements:
* Ensure Rakefile has Appropriate Tasks
* Gemfile should use Test Groups
* Ensure spec_helper.rb is Configured Correctly
* Ensure Asset Routes are Set Correctly
* Lack of model tests
* Feature tests should be declarative, not imperative
* Delete empty helper files
* Use nested forms where appropriate
* No need to set instance vars when redirecting
* Lack of connection between models
* Feature tests should use 'within' for precision
* Use partials appropriately
* Replace instance variables with local variables
* Prefer other Semantic HTML Elements to Divs Where Possible
* Avoid anonymous javascript functions

### Additional comments:
Used bootstrap

No JS


see https://github.com/makersacademy/instagram_challenge/blob/master/docs/review.md for more details on the items above

**Pull request closed without merging**